Dockerfile

```
@@ -1,19 +1,21 @@
-FROM node:lts-alpine
+FROM node:lts-alpine AS build-step
 
 ARG catalogURL
 
-RUN npm install -g serve
-
 WORKDIR /app
-
 COPY package*.json ./
-
 RUN npm install
-
 COPY . .
-
 RUN npm run build -- --catalogUrl=$catalogURL
```

```suggestion
RUN npm run build:minimal -- --catalogUrl=$catalogURL
```

This could even reduce the build size further, if you don't need source maps.

---

Keeping the source map as they actually may become useful for debugging, e.g. for issues with #349